### PR TITLE
gobackup 2.0.0

### PR DESCRIPTION
**Pull request by huacnlee**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

GoBackup v2.0.0 has includes a Web UI, so build script need depends on Node.js and Yarn to prepare `yarn build` first, and then run `go build`.

https://github.com/gobackup/gobackup/releases/tag/v2.0.0

### CODE REVIEW

**Comment by chenrui333** [comment]
no need to touch `bottle` block as it would be automatically updated.

**Comment by huacnlee** [comment]
Okay, I get it.

File: Formula/gobackup.rb [hunk 1]
```diff
@@ -17,11 +16,18 @@ class Gobackup < Formula
...
   def install
     revision = build.head? ? version.commit : version
 
-    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{revision}")
+    chdir "web" do
+      system "yarn", "install"
+      system "yarn", "build"
+    end
+    system "go", "build", "-o", bin/"gobackup", "-ldflags",
+      "-s -w -X main.version=#{revision}"
```
Comment (SMillerDev) [inline Formula/gobackup.rb:30]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{revision}")
```

File: Formula/gobackup.rb [hunk 2]
```diff
@@ -1,10 +1,9 @@
 class Gobackup < Formula
   desc "CLI tool for backup your databases, files to cloud storages"
   homepage "https://gobackup.github.io"
-  url "https://github.com/gobackup/gobackup/archive/v1.6.5.tar.gz"
-  sha256 "8fac65f448f6fa75b13984a71d8af94c59c390f6a64b99c58376014b6996b73c"
+  url "https://github.com/gobackup/gobackup/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "984504dd85e882fe25c02e09f7fcbefa909da349ce2c16c865c6bff65dbef048"
   license "MIT"
-  head "https://github.com/gobackup/gobackup.git", branch: "main"
```
Comment (SMillerDev) [inline Formula/gobackup.rb:7]: Why remove this?

Comment (huacnlee) [reply Formula/gobackup.rb:7]: A mistake, I have updated.

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**